rust/main/agents/relayer/src/msg/op_submitter.rs

```diff
@@ -32,11 +32,13 @@ use hyperlane_core::{
 };
 
 use crate::msg::pending_message::CONFIRM_DELAY;
-use crate::settings::matching_list::MatchingList;
+use crate::server::MessageRetryRequest;
 
 use super::op_queue::OpQueue;
 use super::op_queue::OperationPriorityQueue;
 
+pub const SUBMITTER_QUEUE_COUNT: usize = 3;
```

Comment:
can we add a comment as to why this is important? is there any way we can make sure that e.g. tests fail if this ever becomes stale?

Reply:
Yep, can add comment. Technically there is a way to test this with capacity() and permits
https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Sender.html#method.capacity

But I think that would require us to refactor OpSubmitter in a way that is easily testable via unittests